### DISCUSSION TITLE
Integration with Langserve

### DISCUSSION THREAD

**Discussion by Haxeebraja** [langfuse/langfuse#1431] [Ideas]
### Describe the feature or potential improvement

Langfuse is working fine in langchain.
However, I did not find any example to use langfuse in langserve API. 

Ideally, it should be handled where API is hosted. How to use langfuse in add_routes in langserve?

**Comment by marcklingen** [comment]
If you figure this out, I'd appreciate if you contribute back to the documentation

**Comment by Haxeebraja** [comment]
I was able to work it out with config:

langfuse_handler = CallbackHandler(
    public_key="...",
    secret_key="...",
    host="http://"
)

config = RunnableConfig(callbacks=[langfuse_handler])

chain = conversational_qa_chain.with_config(config)

add_routes(
    app,
    chain,
    path="/chat",
)
